worker-document/src/main/java/com/hpe/caf/worker/document/tasks/AbstractTask.java

```diff
@@ -168,6 +168,11 @@ public void loadScripts() throws DocumentWorkerTransientException, InterruptedEx
         }
     }
 
+    public void unloadScripts() throws DocumentWorkerTransientException, InterruptedException
```

Comment:
I don't see how those exceptions could happen.
```suggestion
    public void unloadScripts()
```